core/src/main/scala/chisel3/experimental/hierarchy/core/Instance.scala

```diff
@@ -132,6 +132,7 @@ object Instance extends SourceInfoDoc {
       )
       class EmptyExtModule extends ExtModule {
         override def desiredName: String = extModName
+        override lazy val name = definition.proto.name
```

Comment:
```suggestion
      class EmptyExtModule extends ExtModule with PseudoModule {
        override def desiredName: String = extModName
```

See the code for `PseudoModule` in the implementation of `final lazy val name`.

Reply:
oh right yes

Reply:
FWIW, I didn't think of that till reviewing your changes 😂